static/src/javascripts/projects/commercial/modules/prebid/bid-config.js

```diff
@@ -323,6 +327,26 @@ const getPangaeaPlacementId = (sizes: PrebidSize[]): number => {
     return 13892409; // Other Section MPU as fallback
 };
 
+const getTripleLiftInventoryCode = (
+    slotId: string,
+    sizes: PrebidSize[]
+): string => {
+    if (containsLeaderboard(sizes))
```

Comment:
Will the linter let you away with an `if` statement without curly brackets!? I always thought we kept them to be consistent, but was sure it was enforced by the linter!

Reply:
I noticed we are not consistent in using braces in 1 liner `if` statement. for example improve adapter uses  braces whereas Xaxis and TrustX don't use braces.